test/test_structured_2d.jl

```diff
@@ -917,6 +917,61 @@ end
         @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 1000
     end
 end
+
+@trixi_testset "elixir_mhd_coupled.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_coupled.jl"),
+                        l2=[
+                            1.0743426980507015e-7, 0.030901698521864966,
+                            0.030901698662039206, 0.04370160129981656,
+                            8.259193827852516e-8, 0.03090169908364623,
+                            0.030901699039770684, 0.04370160128147447,
+                            8.735923402748945e-9, 1.0743426996067106e-7,
+                            0.03090169852186498, 0.030901698662039206,
+                            0.04370160129981657, 8.259193829690747e-8,
+                            0.03090169908364624, 0.030901699039770726,
+                            0.04370160128147445, 8.73592340076897e-9,
+                        ],
+                        linf=[
+                            9.021023431587949e-7, 0.043701454182710486,
+                            0.043701458294527366, 0.061803146322536154,
+                            9.487023335807976e-7, 0.043701561010342616,
+                            0.04370147392153734, 0.06180318786081025,
+                            3.430673132525334e-8, 9.02102342825728e-7,
+                            0.043701454182710764, 0.043701458294525895,
+                            0.06180314632253597, 9.487023254761695e-7,
+                            0.04370156101034084, 0.04370147392153745,
+                            0.06180318786081015, 3.430672973680963e-8,
+                        ],
+                        coverage_override=(maxiters = 10^5,))
+
+    @testset "analysis_callback(sol) for AnalysisCallbackCoupled" begin
+        errors = analysis_callback(sol)
+        @test errors.l2≈[
+            1.0743426980507015e-7, 0.030901698521864966, 0.030901698662039206,
+            0.04370160129981656, 8.259193827852516e-8, 0.03090169908364623,
+            0.030901699039770684, 0.04370160128147447, 8.735923402748945e-9,
+            1.0743426996067106e-7, 0.03090169852186498, 0.030901698662039206,
+            0.04370160129981657, 8.259193829690747e-8, 0.03090169908364624,
+            0.030901699039770726, 0.04370160128147445, 8.73592340076897e-9,
+        ] rtol=1.0e-4
```

Comment:
Two questions because this looks kind of strange:

1. Using `≈` is equivalent to `isapprox.( ... )`, right?
2. Why is there a `rtol=1.0e-4` floating at the end? Is this like passing this as an argument to the `≈` version of `isapprox`?

Reply:
Yes, `≈` should be equivalent to `isapprox`. Using `Test` the syntax is:
`@test 5 ≈ 5.00001 rtol=1e-4`
which is why the `rtol=1.0e-4` is floating at the end.